lib/fast_gettext/translation.rb

```diff
@@ -41,21 +43,24 @@ def n_(*keys, &block)
         block ? block.call : keys.last
       end
     end
+    alias :ngettext :n_
 
-    #translate with namespace, use namespect to find key
+    #translate with namespace, use namespace to find key
     # 'Car','Tire' -> Tire if no translation could be found
     # p_('Car','Tire') <=> s_('Car|Tire')
     def p_(namespace, key, separator=nil, &block)
-      msgid = "#{namespace}#{separator||NAMESPACE_SEPARATOR}#{key}"
+      msgid = "#{namespace}#{CONTEXT_SEPARATOR}#{key}"
```

Comment:
separator is ignored now ?

Reply:
My mistake. I will add separator back and update readme with usage